src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,33 +1,61 @@
 package edu.kis.vh.nursery;
 
+/**
+ * Class which counting down, which is working like a stack.
+ */
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private static final int MIN = -1;
+    private static final int MAX = 11;

```

Comment:
This constant is related to MAX_SIZE_NUMBERS. Changing only one leads to an error (inconsistency).